### DISCUSSION TITLE
Brightness of cursor

### DISCUSSION THREAD

**Discussion by donorport** [MonitorControl/MonitorControl#1352] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Discussion

Is there a way to have the cursor match the brightness? It always stays the same brightness as the display which makes it stand out at night in low light.

**Comment by waydabber** [comment]
This only happens if you use overlay dimming instead of DDC and colorable dimming. With overlay dimming cursor brightness cannot be changed.

**Reply by donorport** [reply]
Can we change from overlay to DDC / Colorable dimming or are you saying your app only controls overlay dimming?
